Gemfile

```diff
@@ -147,6 +147,7 @@ def override_gem(name, *args)
     dependencies.delete(dependency)
 
     gem name, *args
+    Bundler::UI::Shell.new.warn "Caution, using override_gem for #{name} with args: #{args.inspect}, review Gemfile.dev.rb if this is not desired!"
```

Comment:
I would prefer fewer messages in my console for intentional configuration.

Not sure if I want a `<blink>Caution, you are using scissors to cut paper. Review craft supplies if this is not desired!</blink>

Maybe

```ruby
Bundler::UI::Shell.new.warn "NOTE: Gemfile.dev.rb: override_gem :#{name}, #{args.inspect}"
```